ksql-engine/src/main/java/io/confluent/ksql/codegen/SqlToJavaVisitor.java

```diff
@@ -490,30 +490,39 @@ private String visitBooleanComparisonExpression(final ComparisonExpression.Type
     ) {
 
       // For now we just support simple prefix/suffix cases only.
-      String paternString = process(node.getPattern(), true).getLeft().substring(1);
-      paternString = paternString.substring(0, paternString.length() - 1);
+      String patternString = process(node.getPattern(), true).getLeft().substring(1);
```

Comment:
nit - move the quotation-trimming into another function and make patternString final:

```
final String patternString = trimQuotes(process(node.getPattern(), true).getLeft());
```